pdns/xsk.cc

```diff
@@ -82,20 +82,20 @@ struct UmemEntryStatus
   Status status{Status::Free};
 };
 
-LockGuarded<std::unordered_map<uint64_t, UmemEntryStatus>> s_umems;
+LockGuarded<std::map<std::pair<void*, uint64_t>, UmemEntryStatus>> s_umems;
```

Comment:
Ah, I always assumed `std::pair` was something that could be hashed, but it turns out it is not.